lib/rubygems/path_support.rb

```diff
@@ -41,6 +56,18 @@ def initialize(env)
 
   private
 
+  ##
+  # The default home directory.
+  # This function was broken out to accommodate tests in `bundler/spec/commands/doctor_spec.rb`.
```

Comment:
I feel this is the kind of comment that gets easily out of date as code evolves and ends up being confusing. The extraction of this method feels neat anyways, so I'd say we don't need the comment. Also, I don't get the "The default home directory" part.

Reply:
As if to prove your point: This comment is _already_ out of date. 😅 

Literally none of it is true anymore (including the "default home directory" part, which is why you don't get it).

Reply:
Fixed in 6dc41e1.